python_requirements.txt

```diff
@@ -0,0 +1,19 @@
+certifi==2020.6.20
+chardet==3.0.4
+cycler==0.10.0
+idna==2.10
+kiwisolver==1.2.0
+matplotlib==3.3.0
+numpy==1.19.1
+pandas==1.1.0
+Pillow==7.2.0
+pyparsing==2.4.7
+python-dateutil==2.8.1
+pytz==2020.1
+requests==2.24.0
+scipy==1.5.2
+seaborn==0.10.1
+six==1.15.0
+tabulate==0.8.7
```

Comment:
can you check which version of tabulate does the image have now?

Reply:
It is same.